src/server/constants.py

```diff
@@ -53,3 +53,8 @@ class QUERIES:
                                         WHERE date BETWEEN %s AND %s
                                         GROUP BY period_number
                                         ORDER BY period_number"""
+
+    SELECT_DATA: str = "SELECT date, value FROM exchange_rate_history"
+    SELECT_NEWEST_DATA: str = (
```

Comment:
Can these names be changed to something more descriptive, like i.e. `SELECT_ALL_RATE_HISTORY`, `SELECT_ALL_RATE_HISTORY_DESC` etc? These are just suggestions, so feel free to change it or leave it, up2u.

Reply:
Done